dev/app/index.html

```diff
@@ -2,22 +2,18 @@
 <html lang="en">
 
 <head>
-
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
   <title>FCC London Campers</title>
```

Comment:
webpack injects a style tag with the compiled css file using the ExtractTextPlugin. I think it's better this way because the file name can have a hash on it to assist with cache invalidation.